src/concrete/ml/sklearn/base.py

```diff
@@ -1560,6 +1844,8 @@ def from_sklearn_model(
         Returns:
             The FHE-compliant fitted model.
         """
+        # For now we don't use X for quantization, only the thresholds
+        # We could  support import with data as quantizer too
```

Comment:
I am not sure what this comment refers to? Are we changing the way we fit trees?

Reply:
It's legacy comment I believe, I can remove it

Reply:
Thanks